### ISSUE TITLE
Cannot invoke mocks invocation

### ISSUE THREAD

**Issue by shubgene** [microcks/microcks#421]
Hi 

Thanks for building this solution. 

I have just started exploring it , and migrated a couple of services for testing using openapi3 spec to postman collection and using upload on microcks. It all looks good on ui with able to run the test ,however when clicking on mocks invocation for service getting below error in console , nothing in application logs.

main.7366b3dd8fe831b4619f.js:1 ERROR Error: Uncaught (in promise): Error: Cannot match any routes. URL Segment: 'admin/invocations/test-service/v1.0'
Error: Cannot match any routes. URL Segment: 'admin/invocations/test-service/v1.0'
    at t.noMatchError (main.7366b3dd8fe831b4619f.js:1)
    at e.selector (main.7366b3dd8fe831b4619f.js:1)
    at e.error (main.7366b3dd8fe831b4619f.js:1)
    at e._error (main.7366b3dd8fe831b4619f.js:1)
    at e.error (main.7366b3dd8fe831b4619f.js:1)
    at e._error (main.7366b3dd8fe831b4619f.js:1)
    at e.error (main.7366b3dd8fe831b4619f.js:1)
    at e._error (main.7366b3dd8fe831b4619f.js:1)
    at e.error (main.7366b3dd8fe831b4619f.js:1)
    at e._error (main.7366b3dd8fe831b4619f.js:1)
    at T (polyfills.d679eab347439ecd4427.js:1)
    at T (polyfills.d679eab347439ecd4427.js:1)
    at polyfills.d679eab347439ecd4427.js:1
    at e.invokeTask (polyfills.d679eab347439ecd4427.js:1)
    at Object.onInvokeTask (main.7366b3dd8fe831b4619f.js:1)
    at e.invokeTask (polyfills.d679eab347439ecd4427.js:1)
    at t.runTask (polyfills.d679eab347439ecd4427.js:1)
    at g (polyfills.d679eab347439ecd4427.js:1)
    at t.invokeTask (polyfills.d679eab347439ecd4427.js:1)
    at i.useG.invoke (polyfills.d679eab347439ecd4427.js:1)

I am using microcks on k8s cluster and deployed using helm charts.

**Comment by lbroudoux** [comment]
Hey @shubgene!

Thanks for testing out Microcks. The `Mocks invocation` link right there allows you to access invocations statistics. It was implemented in a previous version before we did migrate to a new version of UI Framework and Angular. So right now the feature is still missing and you cannot access invocation statistics visualization ...

I forgot that was missing and will create a new issue for tracking that one.
